frontend/src/metabase-lib/lib/Question.ts

```diff
@@ -1274,7 +1274,7 @@ class QuestionInner {
     if (this.isStructured()) {
       const questionWithParameters = this.setParameters(parameters);
 
-      if (this.query().isEditable()) {
+      if (!this.query().readOnly()) {
```

Comment:
`isEditable` is false if there is no metadata for the query, but I think it should not be necessary just for creating a link